dashboard/app/models/pd/enrollment.rb

```diff
@@ -8,6 +8,9 @@
 #  email          :string(255)      not null
 #  created_at     :datetime
 #  updated_at     :datetime
+#  district_name  :string(255)
```

Comment:
yeah i bet this could be a school_district_id foreign keyed to the incoming school_district table's id column soon.